msgraph-developer-proxy/ProxyCommandHandler.cs

```diff
@@ -68,9 +68,9 @@ public class ProxyCommandHandler : ICommandHandler {
 
     }
 
-    private ProxyConfiguration Configuration { get => ConfigurationFactory.Value; }
+    public static ProxyConfiguration Configuration { get => ConfigurationFactory.Value; }
```

Comment:
I made the configuration public and static because we need to pass it to the ConsoleLogger which is instantiated before anything else.